examples/layer-browser/src/examples/sample-layers.js

```diff
@@ -10,7 +10,8 @@ const S2LayerExample = {
     opacity: 0.6,
     getS2Token: f => f.token,
     getPath: f => f.path,
-    getFillColor: f => [f.value * 256, (1 - f.value) * 256, (1 - f.value) * 128],
+    getStrokeColor: f => [255, 255, 128],
```

Comment:
Not sure what happened to my comments, but we should keep the feature dependent fill color or we lose the ability to show neighbouring S2 cells, which is a key part of the demo.

We need to fix this in another place